### PR TITLE
Plateau control

### PR DESCRIPTION
**Pull request by baptistejamin**
Those changes allow updating plateau pressure based on the controls.

### CODE REVIEW

File: src/software/firmware/srcs/pressure_controller.cpp [hunk 1]
```diff
@@ -436,6 +439,53 @@ void PressureController::exhale() {
 
 void PressureController::updateDt(int32_t p_dt) { m_dt = p_dt; }
 
+void PressureController::updatePeakPressure() {
+    int increase = 5u;
+
+    // Only if plateau pressure has been computed
+    if (m_plateauPressure > 0 && m_plateauPressure < UINT16_MAX) {
```
Comment (dsferruzza) [inline src/software/firmware/srcs/pressure_controller.cpp:446]:
```suggestion
    if ((m_plateauPressure > 0) && (m_plateauPressure < UINT16_MAX)) {
```

**Comment by baptistejamin** [comment]
I reworked with your suggestions!

**Comment by jabby** [comment]
Why do you closed this PR? Is it replaced by #182 ?

**Comment by dsferruzza** [comment]
He told me there were regressions. So yes I guess this new one is better! I (we?) will try it tomorrow

**Pull request closed without merging**